lib/actions-oclif/apps.ts

```diff
@@ -64,12 +63,13 @@ export default class AppsCmd extends Command {
 		const balena = getBalenaSdk();
 
 		// Get applications
-		const applications: ExtendedApplication[] = await balena.models.application.getAll(
-			{
-				$select: ['id', 'app_name', 'slug', 'device_type'],
-				$expand: { owns__device: { $select: 'is_online' } },
+		const applications = (await balena.models.application.getAll({
+			$select: ['id', 'app_name', 'slug'],
+			$expand: {
+				is_for__device_type: { $select: 'slug' },
+				owns__device: { $select: 'is_online' },
 			},
-		);
+		})) as ExtendedApplication[];
```

Comment:
I think something went wrong somewhere in the PR, because the device type column is blank for the `apps` command:
```
$ ./bin/balena apps
ID      APP NAME         SLUG                            DEVICE TYPE ONLINE DEVICES DEVICE COUNT
1280646 balena-on-balena unicorn/balena-on-balena                    1              41
1461407 bbb-test         gh_paulo_castro/bbb-test                    0              0
1148412 essentials       page/essentials                             0              0
...
```
Whereas with v12.10.2:
```
$ ./balena-cli-12.10.2/balena apps
ID      APP NAME         SLUG                            DEVICE TYPE           ONLINE DEVICES DEVICE COUNT
1280646 balena-on-balena unicorn/balena-on-balena        intel-nuc             1              41
1461407 bbb-test         gh_paulo_castro/bbb-test        beaglebone-green-wifi 0              0
1148412 essentials       page/essentials                 raspberrypi3          0              0
...
```

Reply:
Btw, `balena apps` is the only command I tested, so (similar) issues may be affecting other commands too.

Reply:
I'm pretty sure this is the only one, I did go through looking for cases that indirectly use `device_type` like the
```ts
getVisuals().table.horizontal(applications, [
	...
	'device_type',
```
below before for this reason, and going back through them again it looks like this is the only one I missed. It's tricky though because `resin-cli-visuals` has no type checking whatsoever